index.html

```diff
@@ -6090,100 +6090,100 @@ <h2 id="AudioWorklet-Examples">
-        </h2>
-        <p>
-          This is an <code>Event</code> object which is dispatched to
-          <a><code>ScriptProcessorNode</code></a> nodes. It will be removed
-          when the ScriptProcessorNode is removed, as the replacement
-          <a>AudioWorkletNode</a> uses a different approach.
-        </p>
-        <p>
-          The event handler processes audio from the input (if any) by
-          accessing the audio data from the <code>inputBuffer</code> attribute.
-          The audio data which is the result of the processing (or the
-          synthesized data if there are no inputs) is then placed into the
-          <code>outputBuffer</code>.
-        </p>
-        <dl title=
-        "[Constructor((DOMString type, AudioProcessingEventInit eventInitDict)]interface AudioProcessingEvent : Event"
-        class="idl">
-          <dt>
-            readonly attribute double playbackTime
-          </dt>
-          <dd>
-            <p>
-              The time when the audio will be played in the same time
-              coordinate system as the <a><code>AudioContext</code></a>'s
-              <a href="#widl-BaseAudioContext-currentTime">currentTime</a>.
-            </p>
-          </dd>
-          <dt>
-            readonly attribute AudioBuffer inputBuffer
-          </dt>
-          <dd>
-            <p>
-              An AudioBuffer containing the input audio data. It will have a
-              number of channels equal to the
-              <code>numberOfInputChannels</code> parameter of the
-              createScriptProcessor() method. This AudioBuffer is only valid
-              while in the scope of the <a href=
-              "#widl-ScriptProcessorNode-onaudioprocess"><code>onaudioprocess</code></a>
-              function. Its values will be meaningless outside of this scope.
-            </p>
-          </dd>
-          <dt>
-            readonly attribute AudioBuffer outputBuffer
-          </dt>
-          <dd>
-            <p>
-              An AudioBuffer where the output audio data should be written. It
-              will have a number of channels equal to the
-              <code>numberOfOutputChannels</code> parameter of the
-              createScriptProcessor() method. Script code within the scope of
-              the <a href=
-              "#widl-ScriptProcessorNode-onaudioprocess"><code>onaudioprocess</code></a>
-              function is expected to modify the <code>Float32Array</code>
-              arrays representing channel data in this AudioBuffer. Any script
-              modifications to this AudioBuffer outside of this scope will not
-              produce any audible effects.
-            </p>
-          </dd>
-        </dl>
-        <section>
-          <h3>
-            AudioProcessingEventInit
-          </h3>
-          <dl title="dictionary AudioProcessingEventInit : EventInit" class=
-          "idl">
+        <section class="informative">
```

Comment:
I think this is what we want: https://rawgit.com/WebAudio/web-audio-api/9c13639f811292d6570e1506c060a5bbe38d73e0/index.html#the-scriptprocessornode-interface---deprecated

I think this looks right, and tidy doesn't complain about mismatched tags.